### ISSUE TITLE
NVDA announces same information twice after activating "accepting an alternative form" button.

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#15202]
Test Environment:
OS: Windows 11 version 22H2 (os build 22621.1992)
Browser: Chrome 115.0.5790.110
Screen Reader: NVDA version 2023.1

User Impact:
Users who rely on screen readers will be impacted if NVDA announces same information twice after activating "accepting an alternative form?" button.

Repro Steps:
1. Open the URL https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/examples/dialog/ in latest chrome browser.
2. Navigate to 'Add Delivery Address' control present in Example section of the page using down arrow key and invoke it using enter key.
3. Add Delivery Address dialog will appear.
4. Navigate to Verify address control in the dialog using down arrow key and invoke it using enter key.
5. Verification Result dialog will appear.
6. Navigate to 'Accepting an alternative form' button using down arrow key and invoke it using enter key.
7. Verify whether NVDA is announcing same information twice or not.

Actual Result:
NVDA announces same information twice after activating "accepting an alternative form" button.

Expected Result:
NVDA should not announce same information twice after activating "accepting an alternative form" button.
<img width="936" alt="NVDA announces same information twice after activating accepting an alternative form  button" src="https://github.com/nvaccess/nvda/assets/93735775/be122f91-02a1-4d92-ba16-8ecb53106da9">

**Comment by seanbudd** [comment]
We are unable to process this issue as it stands.

Please fill out the issue template properly.

Please confirm that the issue is specifically that the modal dialog title is read twice "End of the road".
